plugins/CoreHome/angularjs/reporting-menu/reportingmenu.controller.js

```diff
@@ -219,6 +223,29 @@
             enterSubcategory(found.category, found.subcategory, found.subsubcategory);
         });
 
+        function showOnlyRawDataMessage() {
+            notification.remove(notificationId);
+
+            menuModel.fetchVisits(decodeURIComponent(date), period, function(json) {
+                if (json.value > 0) {
+                    return;
+                }
+
+                menuModel.fetchLiveVisits(decodeURIComponent(date), period, function(json) {
+                    if (json.length == 0) {
+                        return;
+                    }
+
+                    var url = piwik.broadcast.getHashFromUrl();
+                    url = piwik.broadcast.updateParamValue('category=General_Visitors', url);
+                    url = piwik.broadcast.updateParamValue('subcategory=Live_VisitorLog', url);
```

Comment:
You should replace this with
```suggestion
                    var url = broadcast.buildReportingUrl('category=General_Visitors&subcategory=Live_VisitorLog')
```
otherwise it might contain unwanted parameters.

Reply:
much nicer 👍